src/codegen/irgen/irgenerator.cpp

```diff
@@ -1073,6 +1073,10 @@ class IRGeneratorImpl : public IRGenerator {
         return rtn;
     }
 
+    ConcreteCompilerVariable* getEllipsis() {
+        llvm::Constant* ellipsis = embedRelocatablePtr(Ellipsis, g.llvm_value_type_ptr, "cEllipsis");
+        return new ConcreteCompilerVariable(typeFromClass(type_cls), ellipsis, false);
```

Comment:
@kmod i fix it as you said that, but i user `Ellipsis->cls` rather than `ellipsis_cls`.